var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -106,9 +114,21 @@ def patch(self):
                     # and using LD_LIBRARY_PATH throughout Spack.
                     f.write('[mkl]\n')
                     f.write('mkl_libs=%s\n'     % 'mkl_rt')
+                    write_library_dirs(f, dirs)
                 elif '^atlas' in spec:
                     f.write('[atlas]\n')
                     f.write('atlas_libs=%s\n'   % names)
+                    write_library_dirs(f, dirs)
+                elif '^netlib-lapack' in spec:
+                    # netlib requires blas and lapack listed
+                    # separately so that scipy can find them
+                    if spec.satisfies('+blas'):
+                        f.write('[blas]\n')
+                        f.write('blas_libs=%s\n' % (names))
+                    if spec.satisfies('+lapack'):
+                        f.write('[lapack]\n')
+                        f.write('lapack_libs=%s\n' % (names))
+                    write_library_dirs(f, dirs)
```

Comment:
This doesn't need to be run for both `[blas]` and `[lapack]` sections? If so, the function is no longer necessary, and we can run this a single time after the if-elif-else block.

Reply:
Oops. You're right. Trying to do too many things at the moment.